resources/assets/less/app.less

```diff
@@ -384,6 +384,23 @@ a.logo.no-hover a:hover {
   background-color: transparent;
 }
 
+.index-blocks {
+  color: var(--text-sub);
+  max-width: 400px;
+  max-height: 15px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-align: center;
+  white-space: nowrap;
```

Comment:
Something in here, probably `max-height` and `overflow`, cut off the bottom of some of the text:

![image](https://github.com/user-attachments/assets/5d7245d8-6ac8-450e-b742-9da2db768111)

Reply:
i didn't notice that. to be honest i think i guessed on the max height. there was some artistry that went into this aha

Reply:
I don’t think and if this CSS should be changing any colors, just the css I mentioned above